routing/css/valhalla.css

```diff
@@ -30,7 +30,7 @@ hr {
 }
 
 div#route_inst {
-  height: 50vh;
+  height: auto;
```

Comment:
This was making info window unnecessary long, so I edited height as auto, please check this doesn't make any problem @kdiluca